packages/experimental/Avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,176 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { Image, NativeModules, ViewProps } from 'react-native';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+import { getMemoCache } from '@fluentui-react-native/framework';
+
+const cache = getMemoCache();
+
+const avatarName = 'MSFAvatarView';
```

Comment:
I would disambiguate the avatarName from the native component name. If this is Avatar, it makes sense for the name to be 'Avatar' from the perspective of the displayName and theme lookup.